packages/block-library/src/paragraph/block.json

```diff
@@ -32,6 +32,11 @@
 		"lightBlockWrapper": true,
 		"__experimentalColor": true,
 		"__experimentalLineHeight": true,
-		"__experimentalFontSize": true
+		"__experimentalFontSize": true,
+		"__experimentalFeatures": {
```

Comment:
I'd remove this `__experimentalFeatures` level, but I don't think this PR should be blocked by that. We can always iterate as more things are moved into this.

Reply:
We can do it later once we settle on the final shape. For now, it makes it clear what is integrated with the block editor features.